@xen-orchestra/lite/src/components/AppHeader.vue

```diff
@@ -8,9 +8,14 @@
     <slot />
     <div class="right">
       <PoolOverrideWarning as-tooltip />
-      <UiButton v-if="isDesktop" :left-icon="faDownload" @click="openXoaDeploy">
-        {{ $t('deploy-xoa') }}
-      </UiButton>
+      <template v-if="isDesktop">
+        <UiButton v-if="xoaFound" :left-icon="faArrowUpRightFromSquare" @click="openXoa">
```

Comment:
It's not a problem in this case, but to prevent possible side effects, we add brackets to the click handlers:
```suggestion
        <UiButton v-if="xoaFound" :left-icon="faArrowUpRightFromSquare" @click="openXoa()">
```